README.md

```diff
@@ -42,14 +47,25 @@ docker run -d -p 8787:8787 robinlovelace/itsleeds
 
 Then open a browser and navigate to http://localhost:8787/ (or if you're using Windows or Mac, follow [these instructions](https://github.com/rocker-org/rocker/wiki/Using-the-RStudio-image)).
 
-Note: if you want to install your docker files remotely, e.g. to your an 'extra' folder on your computer,
+
+### Optional techie stuff
+
+If you want to install your docker files remotely, e.g. to your an 'extra' folder on your computer,
 edit the relevant file (e.g. `/etc/default/docker` on Ubuntu) so it becomes, as documented
 [here](https://forums.docker.com/t/how-do-i-change-the-docker-image-installation-directory/1169):
 
+The Dockerfile defining itslive is located here: https://github.com/ITSLeeds/ITS-Live/blob/master/docker/Dockerfile
+
 ```
 # Use DOCKER_OPTS to modify the daemon startup options.
 DOCKER_OPTS="--dns 8.8.8.8 --dns 8.8.4.4 -g /home/robin/extra/docker"
 ```
+## Installation via local .deb package
```

Comment:
Good to have that there also as a placeholder. Right to go with Docker 1st IMO as its fastest to get something up and running with.